### ISSUE TITLE
Built.io discussion

### ISSUE THREAD

**Issue by relatedcode** [relatedcode/ParseAlternatives#33]

**Comment by rihadavid** [comment]
I decided to migrate my Xamarin app from Parse about two weeks ago and I did a little research. 

From what I've found, built.io seems to be the most similar and easy to migrate mbaas, it offers almost all the functionality that parse does, eg. core, push, analytics, storage, ACL, increment on number fields, array fields, pointer fields with include, cloud code functions and beforeSave, ... Only thing I don't like is the dashboard which is not much user friendly when it comes to managing data in tables, but it seems easy to reimplement if users ask for it.

They also have quite good support (if you want an account, you are going to have a call with director of sales) and if you have any trouble, there is a chat with built.io on the dashboard.

They aim for enterprise apps and are offering annual plans based on usage (API calls, storage etc) with quarter-year billing. They are offering student accounts with some free services, however they did not activate this for me yet.

My biggest concern about migrating to built.io is how reliable it is in the long term. They have about 180 developers, which sounds good, but almost all of them are in India.

"The company Built.io is profitable and is 100% customer funded. There is no outside funding. We are here for the long haul." [said the CTO](https://www.quora.com/What-alternatives-exist-to-Parse-now-that-it-is-shutting-down) in the response to Parse shutting down.

But still, I don't know anything about how it works in India and I don't know if I can be sure that the Backend will be running for many years. What if some company acquires built.io and close it just like Facebook did with Parse? What if MBaaS market will no longer be profitable and they close the bussiness? They don't have such a big community as Parse had so I don't know whether they would offer some migration options if it comes to closing the business or just stopped everything.
After the situation with Parse, I am kind of afraid to put my business in hands of MBaaS and lock in and the fact that built.io is from India makes me little nervous as I have no experience with Indian market.

Thanks for any opinions :)

**Comment by nishantpatel** [comment]
Hi David,

Thank you for the review of Backend. I completely agree with your assessment and understand your concerns and I wanted to make sure you saw my blog post today about the subject. I think this will address most of your concerns:

[https://www.built.io/blog/2016/02/the-no-bullshit-take-on-mbaas-and-our-offer-to-the-parse-community]

Regarding our engineers in India, I wanted to clarify that this isn’t an outsourced development shop. All our engineers are employees of Built.io with strong leadership roles including architects and engineering leads. Without them, we wouldn’t have built three award-winning innovative products (a mobile backend, [a headless CMS](https://www.built.io/products/contentstack/overview), and a [cloud-based enterprise integration platform](https://www.built.io/products/flow/features)). Our US team is focused more on business needs such as marketing, sales, etc.

Thanks,
Nishant

**Comment by Bizzi-Body** [comment]
I love that built.io blog post.  No BS.

Anyhow the key lines in that post are:

> Built.io Backend however is not a good fit for:
> Single apps with no plan to scale or grow your user base significantly.
> Developers or companies who are looking for a free Parse replacement.

Which says - for me as an indie - that likely not a good alternative to Parse.  Move along nothing to see here.

**Comment by frgmt** [comment]
Yeah, and that's pretty sad while it seems built.io is the only mbaas with an up to date ios-swift-sdk out there right now.
